src/app/ipv/middleware.js

```diff
@@ -228,16 +232,12 @@ function handleAppStoreRedirect(req, res, next) {
   const specifiedPhoneType = req.params.specifiedPhoneType;
 
   try {
-    switch (specifiedPhoneType) {
-      case PHONE_TYPES.IPHONE:
-        res.redirect(APP_STORE_URL_APPLE);
-        break;
-      case PHONE_TYPES.ANDROID:
-        res.redirect(APP_STORE_URL_ANDROID);
-        break;
-      default:
-        throw new Error("Unrecognised phone type: " + specifiedPhoneType);
+    if (specifiedPhoneType === PHONE_TYPES.IPHONE) {
+      res.redirect(APP_STORE_URL_APPLE);
+    } else if (specifiedPhoneType === PHONE_TYPES.ANDROID) {
+      res.redirect(APP_STORE_URL_ANDROID);
     }
```

Comment:
Careful - I think this reintroduces the same bug that we used to have. You need to return from each of these blocks or the control flow will continue through to the error.

I'd suggest that we do `return res.redirect(...)` so we don't fall through to any other branches

(I also think the switch is more readable, unless there was a particular reason to change it?)